client/src/partials/subsidy/subsidy.html

```diff
@@ -0,0 +1,151 @@
+<header data-header>
+  {{ "SUBSIDY.CREATE_TAXE" | translate }}
+</header>
+
+<nav>
+  <div class="pull-left">
```

Comment:
Since this module is only a single page, it doesn't make much sense to have breadcrumbs.  What function does it serve?